SayTheirNames/Source/Utils/Theme/Theme.swift

```diff
@@ -124,6 +124,9 @@ enum Theme {
                 
             }
         }
-        
+        // Donation Details
+        enum DonationDetails {
+            static let horizontalPadding: CGFloat = 25
```

Comment:
should't PersonController have the same padding with this screen?

Reply:
I think at the time I couldn't count well or the design had different padding - that might be a task for a followup PR

Reply:
will you create a card for it or should I do it?